scripts/parse_pricing_api.py

```diff
@@ -0,0 +1,303 @@
+"""Script that parses pricing data from Google's API.
```

Comment:
Can you find some way to rewrite this as node.js or shell script? I'm really, really, really hesitant to introduce python to this repo.

Reply:
Including the script in this PR isn't **necessary**, but I thought I'd add it here as a "show your work" kind of deal. I did make an attempt with Bash but it really ended up taking me too much effort for something that isn't necessary.

What I can do is host the Python script elsewhere, and add a pointer here to that script. How does that sound?

Reply:
Would you consider just using node?

Reply:
Sure, the script can be any language. I used Python just because I have more expertise in it. Re-writing this in Node would take me a while, though not impossible. 

For now I'll remove this script from the PR and put it somewhere else. I'll add a TODO task to re-write the script in Node for anyone to do.

Reply:
I mentioned this in a code comment and the PR description but I have the script as a gist here: https://gist.github.com/wnojopra/be5be3ab7e6a09a351b89085992a940d

Reply:
@wnojopra could you please add the script to [dsp-scripts/terra-ui](https://github.com/broadinstitute/dsp-scripts/tree/master/terra-ui)? Sorry for the trouble.

Reply:
Done, https://github.com/broadinstitute/dsp-scripts/pull/96 .

Reply:
I'll review that PR separately.